packages/store-api/src/platforms/vtex/resolvers/aggregateRating.ts

```diff
@@ -0,0 +1,7 @@
+type Resolver = () => unknown
+
+// TODO: Add a review system integration
+export const StoreAggregateRating: Record<string, Resolver> = {
+  ratingValue: () => 5,
+  reviewCount: () => 0,
+}
```

Comment:
hmmm, I'm in doubt whether we should have a file for a resolver that doesn't exist.